zero_bin/prover/src/lib.rs

```diff
@@ -20,9 +22,11 @@ use tracing::info;
 use zero_bin_common::fs::generate_block_proof_file_name;
 
 #[derive(Debug, Clone, Copy)]
-pub struct ProverParams {
+pub struct ProverConfig {
```

Comment:
Do we actually need to have a `ProverConfig` anymore, given that it is semantically identical to `CliProverConfig`, and that all fields are guaranteed to be initialized?

Reply:
@Nashtare I don't much like the idea of `clap` structure (`CliProverConfig`) being passed as an argument deep in the stack, but yeah, we can remove `ProverConfig`.